packages/schematics/angular/utility/project.ts

```diff
@@ -35,6 +43,23 @@ export function getProject<TProjectType extends ProjectType = ProjectType.Applic
   return workspace.projects[projectName] as WorkspaceProject<TProjectType>;
 }
 
+export function getProjectConfig(host: Tree, path: string) {
```

Comment:
getProjectTsConfig Maybe? As config is more broad and might imply workspace project config,

Reply:
Yes, totally makes sense. :+1:

Reply:
Most likely actually this method will become redundant.

Reply:
Indeed